build/helper/metadata_add_all.py

```diff
@@ -237,6 +237,33 @@ def test_add_all_metadata_simple():
                 'description': 'Performs a foo, and performs it well.',
             },
         },
+        'private_method': {
+            'codegen_method': 'private',
+            'returns': 'ViStatus',
+            'parameters': [
+                {
+                    'direction': 'in',
+                    'enum': None,
+                    'name': 'vi',
+                    'type': 'ViSession',
+                    'documentation': {
+                        'description': 'Identifies a particular instrument session.',
+                    },
+                },
+                {
+                    'direction': 'out',
+                    'enum': None,
+                    'name': 'status',
+                    'type': 'ViString',
+                    'documentation': {
+                        'description': 'Return a device status',
+                    },
+                }
+            ],
+            'documentation': {
+                'description': 'create a private method',
```

Comment:
This is the description of the function. While functionally it makes no difference, you should describe the function not what you're doing in the PR, as it could confuse someone.